sklearn_numba_dpex/common/kernels.py

```diff
@@ -304,6 +316,53 @@ def partial_sum_reduction(
         if first_work_id:
             add_first_cols(local_values, result, group_id)
 
+    return partial_sum_reduction
+
+
+@lru_cache
+def make_sum_reduction_2d_axis1_kernel(
+    size0, size1, work_group_size, device, dtype, fused_unary_func=None
```

Comment:
The changes to this sum are the addition of this arg `fused_unary_func` which enable fusing a unary ops on the elements of the input before summing. It's used in this PR to compute variance.